.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+    - package-ecosystem: gradle
+      directory: "/"
+      schedule:
+        interval: "weekly"
```

Comment:
I think we need to add `open-pull-requests-limit` here as well. More info [here](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#open-pull-requests-limit).

Reply:
I saw it, but why do you think we need it?

Reply:
To avoid cluttering the opened PRs while not processing them.
Also, with every merge on the `main` branch, the bot will try to rebase the branches with the opened PRs, which will lead to CI congesting.

Reply:
I don't think we will have that problem here, the number of dependencies we have is very limited

Reply:
I see.
Anyways I think the default will work just fine.
> By default, Dependabot opens a maximum of five pull requests for version updates.